### PR TITLE
Windows ami Integration tests updates

### PR DESCRIPTION
**Pull request by hanumantharaomvl**
##### SUMMARY
This PR contains the following fixes.

1. Windows AMI Update for Integration Tests.
2. Windows Integration tests updates.

Validated the changes.

### CODE REVIEW

File: test/integration/targets/connection/test_connection.yml [hunk 1]
```diff
@@ -29,8 +29,6 @@
 
   - name: remove local file with unicode filename and content
     local_action: file path={{ local_tmp }}-汉语/汉语.txt state=absent
-  - name: fetch remote file with unicode filename and content
-    fetch: src={{ remote_tmp }}-汉语/汉语.txt dest={{ local_tmp }}-汉语/汉语.txt fail_on_missing=true validate_checksum=true flat=true
```
Comment (psharkey) [inline test/integration/targets/connection/test_connection.yml:33]: What's the reasoning behind removing this task?

File: test/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup/tasks/main.yml [hunk 2]
```diff
@@ -100,7 +100,6 @@
       template:
         dest: "{{playbook_dir}}/aws-env-vars.sh"
         src: aws-env-vars.j2
-      no_log: yes
```
Comment (psharkey) [inline test/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup/tasks/main.yml]: Won't this leak credentials to the console now?

File: test/integration/targets/connection_aws_ssm/aws_ssm_integration_test_teardown/tasks/main.yml [hunk 3]
```diff
@@ -6,7 +6,6 @@
       aws_secret_key: "{{aws_secret_key}}"
       region: "{{aws_region}}"
       security_token: "{{security_token}}"
-  no_log: true
```
Comment (psharkey) [inline test/integration/targets/connection_aws_ssm/aws_ssm_integration_test_teardown/tasks/main.yml]: Same as above - looks like this exposes credentials now.

**Review by psharkey — CHANGES_REQUESTED** [review]

**Comment by hanumantharaomvl** [comment]
Changes were already checked-in. Closing the PR.

**Pull request closed without merging**